hands-on-exercise/2-ignite-cli-adv/3-game-fifo.md

```diff
@@ -323,6 +323,24 @@ You have implemented a FIFO that is updated but never really used. It will be us
 
 At this point, your previous unit tests are failing, so they must be fixed. Add `FifoHeadIndex` and `FifoTailIndex` in your value requirements on `SystemInfo` as you [create games](https://github.com/cosmos/b9-checkers-academy-draft/blob/game-fifo/x/checkers/keeper/msg_server_create_game_test.go#L46-L47) and [play moves](https://github.com/cosmos/b9-checkers-academy-draft/blob/game-fifo/x/checkers/keeper/msg_server_play_move_test.go#L98-L99). Also add `BeforeIndex` and `AfterIndex` in your value requirements on `StoredGame` as you [create games](https://github.com/cosmos/b9-checkers-academy-draft/blob/game-fifo/x/checkers/keeper/msg_server_create_game_test.go#L60-L61) and [play moves](https://github.com/cosmos/b9-checkers-academy-draft/blob/game-fifo/x/checkers/keeper/msg_server_play_move_test.go#L112-L113).
 
+Edit `types/genesis_test.go`, so that it incorporates the newly added fields `FifoHeadIndex` and `FifoTailIndex` with the correct initialization values.
```

Comment:
```suggestion
Edit `types/genesis_test.go`, so that it incorporates the newly added fields `FifoHeadIndex` and `FifoTailIndex` with the correct initialization values:
```

Reply:
added this as well